Parser/pegen/pegen.h

```diff
@@ -261,11 +261,23 @@ int _PyPegen_check_barry_as_flufl(Parser *);
 mod_ty _PyPegen_make_module(Parser *, asdl_seq *);
 
 // Error reporting helpers
-expr_ty _PyPegen_get_invalid_for_target(expr_ty e);
 expr_ty _PyPegen_get_invalid_target(expr_ty e, int del_targets);
 #define GET_INVALID_TARGET(e) _PyPegen_get_invalid_target(e, 0)
 #define GET_INVALID_DEL_TARGET(e) _PyPegen_get_invalid_target(e, 1)
 
+Py_LOCAL_INLINE(expr_ty)
+_PyPegen_get_invalid_for_target(expr_ty e)
```

Comment:
Given that this is mainly for error reporting (so no hot path), I am not sure if it makes sense to force the inlining, no?

Reply:
I liked the idea of having one main visitor, `_PyPegen_get_invalid_target`, which is a function and then build macros around it. This seemed too big for a macro and so I chose an inline function. Do we only use inline functions for speedup? I'm certainly no expert, so it's your call.

Reply:
> Do we only use inline functions for speedup?

Normally is done for performance or for symbol visibility (check the C99 inline visibility rules). I am happy with the function, I just surprised on why you added the `Py_LOCAL_INLINE`.

Reply:
Ok then! Reverting.